warehouse/manage/views.py

```diff
@@ -558,18 +560,20 @@ def manage_project_roles(project, request, _form_class=CreateRoleForm):
                 .join(Role.user)
                 .filter(Role.role_name == 'Owner', Role.project == project)
             )
-            owner_emails = [owner.user.email for owner in owners]
-            owner_emails.remove(request.user.email)
-
-            send_collaborator_added_email(
-                request,
-                user,
-                request.user,
-                project.name,
-                form.role_name.data,
-                owner_emails
-            )
-
+            owner_emails = [owner.user.email for owner in owners
+                            if owner.user.email]
+            if request.user.email in owner_emails:
+                owner_emails.remove(request.user.email)
```

Comment:
We should probably still email the user making the change. This is a relatively major "operation" and if someone has intercepted a password and starts making changes as a user, we'd want them to notify.

Though I it's possible for an attacker to simply remove or replace all active email addresses before making these changes, the primary email address changing would initiate a notification.